Reduce GitHub requests count (#38)
Microsoft/AppCenter-SDK-Unity-Extension: CHANGELOG.md

# App Center SDK for Unity Editor Extensions Change Log

## Development version

> * **[Bug fix]** Fixed excess GitHub API calls from unity extension

Reviewer: ```suggestion
* **[Bug fix]** Fixed excess GitHub API calls from unity extension.
```

> * **[Bug fix]** Fixed excess GitHub API calls from unity extension

Reviewer: ```suggestion
* **[Bug fix]** Fixed excessive amount of GitHub API calls from unity extension.
```

> * **[Bug fix]** Fix call to github on every time when editor enters play mode. 

Reviewer 2: Can we rephrase this? Not sure to understand the wording.

Author: Fixed

> * **[Bug fix]** Fixed excessive amount of GitHub API calls from unity extension.

Reviewer: This is "unity extension" repo, may be just "the extension"?

Reviewer 2: ```suggestion
* **[Bug fix]** Fixed excessive amount of GitHub API calls.
```
